cmd/main.go

```diff
@@ -2,30 +2,70 @@ package main
 
 import (
 	"fmt"
+	mongoDB "github.com/dilyara4949/employees-api/internal/database/mongo"
+	"github.com/dilyara4949/employees-api/internal/database/postgres"
+	"github.com/dilyara4949/employees-api/internal/domain"
+	"github.com/dilyara4949/employees-api/internal/grpc/server"
+	"github.com/dilyara4949/employees-api/internal/repository/postgres/employee"
+
+	mongoemployee "github.com/dilyara4949/employees-api/internal/repository/mongo/employee"
+	mongoposition "github.com/dilyara4949/employees-api/internal/repository/mongo/position"
+	"github.com/dilyara4949/employees-api/internal/repository/postgres/position"
+	pb "github.com/dilyara4949/employees-api/proto"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/reflection"
 	"log"
 	"net"
 	"net/http"
 
 	conf "github.com/dilyara4949/employees-api/internal/config"
 	"github.com/dilyara4949/employees-api/internal/controller"
-	"github.com/dilyara4949/employees-api/internal/grpc/server"
-	"github.com/dilyara4949/employees-api/internal/repository/employee"
-	"github.com/dilyara4949/employees-api/internal/repository/position"
 	"github.com/dilyara4949/employees-api/internal/route"
-	pb "github.com/dilyara4949/employees-api/proto"
-	"google.golang.org/grpc"
-	"google.golang.org/grpc/reflection"
+)
+
+var (
+	posCollection = "positions"
```

Comment:
1. Using globals is generally discouraged due to memory safety (they can be read and written from anywhere, which could lead to errors. 
2. As I mentioned before, this should be configurable with default values.

```
posCollection := os.GetVar
if posCollection == "" {
  posCollection = defaultPosCollection
}
```

Reply:
fixed